packages/stripe-graphql/src/schema/subscriptions.ts

```diff
@@ -0,0 +1,25 @@
+import { builder } from '../builder'
+
+builder.objectType('StripeSubscriptions', {
```

Comment:
These list-wrapping object types would more commonly be represented as relay connection objects.

Reply:
Skipping for now: https://github.com/nhost/nhost/pull/916#discussion_r964431726